ActindoWorkFlow/Overview/02_WorkflowProcessElements.md

```diff
@@ -0,0 +1,194 @@
+
+- **Optional**  
+ Optional ports are displayed with a ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button next to each of them. By clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button a new place is added, which can in turn be linked to a new transition. For detailed information about adding new actions, see [Create an action](/ActindoWorkFlow/Operation/01_ManageWorkflows.md#create-an-action).
+
+
+
+## Actions
+
+![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/AddAction.png "[Action]")
+
+Actions, or process actions, are the individual activities being executed in the workflow process and are represented by a rectangle.
+
+A process action is the actual execution of a transition  configured in a workflow. As such, a process action can be successful or faulty depending on its execution. If an error occurs, a process action can be retried if the underlying cause has been fixed, which results in a re-execution of the process action. Moreover, if multiple tokens move through the workflow, every token is processed in its own process action. As a result, a transition can be associated with multiple process actions, depending of the number of tokens running through it.
+
+
+
+## Tokens
+
+![Tokens](/Assets/Screenshots/ActindoWorkFlow/Workflows/Tokens.png "[Tokens]")
+
+
+A token is a container carrying data needed to execute an action, for example a number or a document.
+
+The *Processes* menu entry displays a diagram of any process taking place in real time and a list of all actions being executed in the selected process. For detailed information about tracking a workflow process, see [Track workflow process](/ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md).
+
+A process is started with a single token at the start place, which will be input in an action to be executed. In turn, an action may output several tokens, which will become inputs for subsequent actions. It is therefore possible that a place contains several tokens, for example, several delivery notes issued for different parts of a same order. Each token will then be processed separately, that means, that the subsequent actions will be executed once for each token available.
+
+
+The workflow process view displays the actions in different colors depending on their current status:
+
+  - *Ready*
+  - *In Progress*
+  - *Error*
+  - *Done*
+  - *Dead*
+  - *Suspended*
+  - *Aborted*
+
+For detailed information about the process actions, see [Process actions](/ActindoWorkFlow/UserInterface/04a_ProcessActions.md).
+
+Next to the status of each action, a number is displayed. This number refers to the number of tokens processed in this action, and it depends on the action status and the number of tokens received. For example, if an action displays the message *Done 1*, this means that the action has already been executed and that 1 token has been processed.
```

Comment:
die zahl wird nur angezeigt bei actions, die entweder den status in progress, done oder error haben -> mindestens 1 token muss schon in die action gelaufen sein.